overlord/managers_test.go

```diff
@@ -2422,3 +2422,81 @@ func (ms *mgrsSuite) TestUpdateWithAutoconnectRetrySlotSide(c *C) {
 func (ms *mgrsSuite) TestUpdateWithAutoconnectRetryPlugSide(c *C) {
 	ms.testUpdateWithAutoconnectRetry(c, "other-snap", "some-snap")
 }
+
+func (ms *mgrsSuite) TestRemoveWithDisconnectIgnore(c *C) {
```

Comment:
what is this testing exactly, I'm not sure I understand from the name

Reply:
This is testing that if snaps on both end of connection are removed, we don't create disconnect hooks for both removals. (i.e. findSymmetricInstallTask returns "ignore"). I admit the test was lacking a stricter check for that, added it now and also renamed the test.